packages/fxa-shared/db/models/auth/paypal-ba.ts

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { AuthBaseModel } from './auth-base';
+
+/**
+ * Tracks current and prior PayPal Billing Agreements for a user.
+ */
+export class PayPalBillingAgreements extends AuthBaseModel {
+  public static tableName = 'paypalCustomers';
+  public static idColumn = 'uid';
+
+  protected $uuidFields = ['uid'];
+
+  public uid!: string;
+  public billingAgreementId!: string;
+  public status!: string;
+
+  public createdAt!: number;
+  public endedAt!: number;
```

Comment:
This can be null in the db.  If we make the type here `number | null`, would the ORM be smart enough to set it to null?  (This depends on how you feel about interpreting 0 as null...assuming the value here is 0 when it's null in the db record.)

Reply:
Ah, good poing!